uptane/services/inventorydb.py

```diff
@@ -169,4 +84,122 @@ def scrub_filename(fname, expected_containing_dir):
   return abs_fname
 
 
+class InventoryDataBase():
+  """
+  Dictionary design:
+
+  {
+    'vehicle manifests':
```

Comment:
Yes, if we are going to use global variables instead of class an dictionary as a class member,
would be better to split them into the some kind of variables